node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
+
+    /**
+     * List of interfaces implemented by the translator.
+     */
+    public readonly interfaceNames: string[];
+
+    /**
+     * Name of the onboarding module required by the translator.
+     */
+    public readonly onboardingName: string;
```

Comment:
ok.. may be onboardingMethodName ?

---
In reply to: [71910673](https://github.com/openT2T/opent2t/pull/1#discussion_r71910673) [](ancestors = 71910673,71904172)